pwndbg/commands/nearpc.py

```diff
@@ -97,6 +99,10 @@ def nearpc(pc=None, lines=None, to_string=False, emulate=False):
 
         line   = ' '.join((prefix, "%#x" % i.address, s or '', asm))
 
+        # Highlight the current line if the config is enabled
```

Comment:
Was that a typo or where did you post screenshots, I can't see any :cry: 

However the currently updated PR now looks like this:

![highlight-source](https://cloud.githubusercontent.com/assets/203012/16063479/e0f8e370-3299-11e6-8efc-b7295aa83909.png)
![highlight-with-color](https://cloud.githubusercontent.com/assets/203012/16063480/e0fb5bd2-3299-11e6-95bb-5cd7cfbf38d5.png)
![highlight-with-colored-data](https://cloud.githubusercontent.com/assets/203012/16063481/e0fde8f2-3299-11e6-85a3-64f02cf29333.png)

It also demonstrates that if at any point in the future the ASM output gets own coloring that those colors will be preserved and in such case only the prefix, address and symbol name will be highlighted. Right now it does color everything that does not have an explicit color.
Does this look OK to you? :smile_cat: